source/features/restore-file.tsx

```diff
@@ -127,7 +123,7 @@ function handleMenuOpening({delegateTarget: dropdown}: DelegateEvent): void {
 			role="menuitem"
 			type="button"
 		>
-			Restore file
+			Drop from PR
```

Comment:
I am not sure i like this wording, we "restoring the file" to how it was prior to the pr.

Reply:
Yeah I suggested a change because "Restore" isn't really clear what we're restoring to.